python/lsst/ts/electrometer/electrometerCSC/ElectrometerCSC.py

```diff
@@ -1,480 +1,734 @@
+
+        # Events declaration
+        self.evt_appliedSettingsMatchStart_data = self.evt_appliedSettingsMatchStart.DataType()
+        self.evt_detailedState_data = self.evt_detailedState.DataType()
+        self.evt_digitalFilterChange_data = self.evt_digitalFilterChange.DataType()
+        self.evt_errorCode_data = self.evt_errorCode.DataType()
+        self.evt_integrationTime_data = self.evt_integrationTime.DataType()
+        self.evt_intensity_data = self.evt_intensity.DataType()
+        self.evt_largeFileObjectAvailable_data = self.evt_largeFileObjectAvailable.DataType()
+        self.evt_measureRange_data = self.evt_measureRange.DataType()
+        self.evt_measureType_data = self.evt_measureType.DataType()
+        self.evt_settingsAppliedReadSets_data = self.evt_settingsAppliedReadSets.DataType()
+        self.evt_settingsAppliedSerConf_data = self.evt_settingsAppliedSerConf.DataType()
+        self.evt_settingVersions_data = self.evt_settingVersions.DataType()
+
+        # Electrometer declarations
+        if(self.simulated == 1):
+            self.electrometer = ElectrometerSimulator()
+        else:
+            self.electrometer = ElectrometerController()
+        self.appliedSettingsMatchStart = False
+        self.detailed_state = iec.ElectrometerStates.OFFLINESTATE
+        self.stop_triggered = False
+        self.lastScanTime = 0
+
+        # Loop initialization
+        asyncio.ensure_future(self.init_stateLoop())
+        asyncio.ensure_future(self.init_intensityLoop())
+
+        # salobj configuration to run the same command mult
+        self.cmd_startScanDt.allow_multiple_commands = True
+        self.cmd_startScan.allow_multiple_commands = True
+        self.cmd_stopScan.allow_multiple_commands = True
+
+    def do_setLogLevel(self):
+        pass
+
+    def do_setSimulationMode(self):
+        pass
+
+    def do_start(self, id_data):
+        """Start the electrometer CSC. It changed the CSC to standby and clear the
+        electrometer controller configuration to start fresh
+        """
+        self.localConfiguration.setSettingsFromLabel(
+            id_data.data.settingsToApply, self.mainConfiguration)
+        self.publish_settingVersions(
+            self.mainConfiguration.getRecommendedSettings())
+
+        self.apply_serialConfigurationSettings()
+
+        self.apply_initialSetupSettings()
+
+        self.publish_appliedSettingsMatchStart(True)
+
+        self.electrometer.updateState(iec.ElectrometerStates.DISABLEDSTATE)
+        super().do_start(id_data)
+        self.log.debug("Start done...")
+
+    def do_enterControl(self, id_data):
+        """Not used
+        """
+        pass
+
+    def do_disable(self, id_data):
+        """Change state to disabled state
+        """
+        super().do_disable(id_data)
+        self.electrometer.updateState(iec.ElectrometerStates.DISABLEDSTATE)
+        self.log.debug("Disable done...")
+
+    def do_standby(self, id_data):

```

Comment:
The override could be done using the provided hooks begin_standby or end_standby.